tsfile/src/test/java/org/apache/iotdb/tsfile/read/filter/IExpressionOptimizerTest.java

```diff
@@ -96,9 +95,8 @@ public void testSeriesOnly() {
       IExpression expression = BinaryExpression
           .and(BinaryExpression.or(singleSeriesExp1, singleSeriesExp2),
               singleSeriesExp3);
-      Assert.assertEquals(true,
-          expression.toString()
-              .equals(expressionOptimizer.optimize(expression, selectedSeries).toString()));
+      Assert.assertEquals(expression.toString(),
+          expressionOptimizer.optimize(expression, selectedSeries).toString());
 
     } catch (QueryFilterOptimizationException e) {
       e.printStackTrace();
```

Comment:
Can you replace this e.print  with logger.error()?

Reply:
Use a Assert.fail() is better?